### PR TITLE
[CDAP-12279] Add Directive entity

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-12279

### CODE REVIEW

File: cdap-ui/app/cdap/components/ResourceCenter/index.js [hunk 1]
```diff
@@ -91,74 +126,37 @@ export default class ResourceCenter extends Component {
...
     ReactDOM.unmountComponentAtNode(wizardContainer);
   }
   getWizardToBeDisplayed() {
-    if (this.state.createStreamWizard) {
+    if (!this.state.activeWizard) {
+      return null;
+    } else if (this.state.activeWizard === 'createStreamWizard') {
       return (
         <StreamCreateWithUploadWizard
           isOpen={this.state.createStreamWizard}
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/ResourceCenter/index.js]: I think this needs to be changed to `isOpen={true}`, 'cause right now the Create Stream wizard doesn't open

Comment (elfenheart) [reply cdap-ui/app/cdap/components/ResourceCenter/index.js]: great catch! 👍

**Comment by tonybach** [comment]
Code LGTM 👍

**Comment by ajainarayanan** [comment]
Talked to @bdmogal. Noticed an oversight we missed and have filed a JIRA: https://issues.cask.co/browse/CDAP-12954.

**Pull request merged**